input/cql/ProfilePatterns.cql

```diff
@@ -268,9 +268,9 @@ define Observation:
   [Observation] O
     where O.status in { 'registered', 'preliminary', 'final', 'amended', 'corrected' }
 
-// QICoreObservationNotDone
-// @profile: http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-observationnotdone
-define ObservationNotDone:
+// QICoreObservationCancelled
+// @profile: http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-observationcancelled
+define ObservationCncelled:
```

Comment:
```suggestion
define ObservationCancelled:
```

Reply:
Fixed typo.  Thanks for finding that.   Commit: b43e6b6